.github/workflows/staging/kusama-taskdef.json

```diff
@@ -35,11 +35,11 @@
             "expression": "attribute:ecs.availability-zone==us-east-2a"
         }
     ],
-    "memory": "3945",
+    "memory": "16041",
```

Comment:
Out of curiosity why those odd numbers and not i.e. `16384`?

Reply:
Some of the 16GB is allocated already to the OS.